packages/slate-editor/src/extensions/paragraphs/components/ParagraphElement.tsx

```diff
@@ -12,12 +12,14 @@ interface Props extends HTMLAttributes<HTMLParagraphElement> {
 }
 
 export function ParagraphElement({ attributes, children, className, element, ...props }: Props) {
+    const align = element.align || Alignment.LEFT;
+
     return (
         <p
             {...attributes}
             {...props}
             className={classNames(styles.ParagraphElement, className)}
-            style={{ textAlign: element.align }}
+            style={{ textAlign: align }}
```

Comment:
Isn't it better to use [CSS Logical Properties](https://caniuse.com/css-logical-props) for `text-align`? Using `text-align: start` is equal to `text-align: left` however, using `start` will ensure the RTL documents can benefit from the same fix.

Reply:
Hm... That's a good point. 
My main idea behind omitting `align` if it matches the default was to play well with i18n and RTL languages.

Changing the model, though, isn't easy.

Reply:
So, based on Ali's point, I think it's better to use `text-align: start` and `text-align: end` for left and right alignment respectively as then it will work properly for RTL languages as well. However, the UI still shows the icons as left/center/right whereas it should probably be the other way around for RTL.

No need to change the model at this point, we just change the text align properties.

The [newsroom](https://ali-newsroom.prezly.com/translation-table) itself renders the alignment properly I believe.

Reply:
Looks like this now, this is left, center and right align respectively:

LTR
![Screenshot 2024-07-08 at 17 24 10](https://github.com/prezly/slate/assets/4209081/4fce249b-5cb4-4013-b409-040735202f75)

RTL
![Screenshot 2024-07-08 at 17 23 36](https://github.com/prezly/slate/assets/4209081/68b7fad0-1ce4-48a3-a9e7-1e25ff7a29c7)

So pretty much looks the same as in production right now, just with the bug resolved.